build/cite/ogcapi-features10/README.md

```diff
@@ -0,0 +1,89 @@
+# OGC API Features - Conformance Test Suite execution
+
+These are the instructions on how to run the CITE Conformance Test Suite for the
+*GeoServer* OGC API Features service.
+
+## Goals
+
+* To run them manually on demand in your local machine
+* To run them while you're debugging GeoServer in your IDE
+* To run them in an automated CI/CD pipeline (Github actions workflow), and to upload the reports somewhere
+
+## Requirements
+
+* A *GeoServer* instance with the `ogcapi-features` extension
+* A data set to run the conformance tests against 
+
+## Test Data:
+
+The datasets included for the default test runs are the vector layers from
+the GeoServer *release** data directory.
+
+### Limited CRS list advertised per FeatureType
+
+Each `FeatureType` to be tested must have a limited set of advertised CRS's where for which
+coordinate reprojection is expected to work. Otherwise the CRS tests will be executed
+against all of them.
+
+For example, querying a collection:
+
+```json
+...
+  "crs": [
+    "http://www.opengis.net/def/crs/OGC/1.3/CRS84",
+    "http://www.opengis.net/def/crs/EPSG/0/4326",
+    "http://www.opengis.net/def/crs/EPSG/0/3857"
+  ],
+  ...
+  ```
+  
+as opposed to advertising all the CRS's available in GeoServer.
+
+In order to automate this, the `release` data directory is configured with `EPSG:4326` and `EPSG:3857` as the
+advertised CRS's directly in the global `WFS` settings.
+  
+## Running
+
+For the most common case, you'll want to run the `ogcapi-features10` suite against a GeoServer instance from the local `.war` build.
+
+First, run 
+
+```
+make war
+```
+to build the `geoserver.war` file from `src/web/app`, and
+
+```
+make build suite=ogcapi-feartures10
+```
+
+To build the GeoServer Docker image that will be exercised. Then run:
+
+```
+make test suite=ogcapi-feartures10
+```
+
+These are the same steps the "Run CITE Tests" github actions workflow will follow.
+
+At the end of each test run, a human readable summary of test failures will be printed out, if any.
+
+### Test a local instance
+
+For development purposes you may want to run the `Start.java` GeoServer launcher from the `gs-web-app`'s test sources.
```

Comment:
I would add something like:

In order to reproduce the Github action CITE tests results, you'll have use the the data directory at <path to the data dir here>

Reply:
done